pkg/session_recording/recordings.jsx

```diff
@@ -394,16 +398,14 @@
 
             for (i = 0; i < entryList.length; i++) {
                 let e = entryList[i];
-                let boot_id = e["_BOOT_ID"];
                 let session_id = e["TLOG_SESSION"];
-                let process_id = e["_PID"];
 
                 /* Skip entries with missing session ID */
                 if (session_id === undefined) {
                     continue;
                 }
+                let id = e['TLOG_REC'];
```

Comment:
Since ID no longer depends on session_id being present, we can move it up to the beginning of the block and the other `let` statements.